src/material/sidenav/sidenav.md

```diff
@@ -209,6 +209,11 @@ Similarly, the `<mat-sidenav-content>` should be given a role based on what it c
 represents the primary content of the page, it may make sense to mark it `role="main"`. If no more
 specific role makes sense, `role="region"` is again a good fallback.
 
+#### Focus management
+Auto Focus within `<mat-drawer>` component is enabled by default, if it's <b>not</b> in 'side' mode. We  can manage it explicitly, using `autoFocus` input property.
```

Comment:
> The sidenav captures focus automatically by default, unless it's in the `side` mode. If you want to disable focus capturing, you can set the `autoFocus` input to `false`.

Reply:
is this obvious, that you need to set autoFocus input to true if it's in side mode?

Reply:
The last sentence could be changed to this to capture it:

> If you want to disable focus capturing completely or turn it on in `side` mode, you can set the `autoFocus` input.

Reply:
> Focus capturing can be enabled or disabled explicitly, using `autoFocus` input
maybe like this? otherwise, I'll update with your suggestion :)

Reply:
Sounds good.